src/reggae/dub/interop/dublib.d

```diff
@@ -55,33 +55,52 @@ struct Dub {
         return _project.packageManager.getPackage(dubPackage, Version(version_));
     }
 
-    DubConfigurations getConfigs(in from!"reggae.options".Options options) {
+    static auto getGeneratorSettings(in Options options) {
+        import dub.compilers.compiler: getCompiler;
+        import dub.generators.generator: GeneratorSettings;
+        import std.path: baseName, stripExtension;
+
+        const compilerBinName = options.dCompiler.baseName.stripExtension;
+
+        GeneratorSettings ret;
+
+        ret.compiler = () @trusted { return getCompiler(compilerBinName); }();
+        ret.platform = () @trusted {
+            return ret.compiler.determinePlatform(ret.buildSettings,
+                options.dCompiler, options.dubArchOverride);
+        }();
+        ret.buildType = options.dubBuildType;
+
+        return ret;
+    }
+
+    DubConfigurations getConfigs(/*in*/ ref from!"dub.platform".BuildPlatform platform) {
```

Comment:
What's with the commented out `in`?

Reply:
The explanation is a few lines down - a missing `in` in dub causes this inconvenience.